site/plugins.json

```diff
@@ -86,7 +86,16 @@
     "name": "Visual diff (Applitools)",
     "package": "netlify-plugin-visual-diff",
     "repo": "https://github.com/applitools/netlify-plugin-visual-diff",
-    "version": "1.2.0"
+    "version": "2.0.0",
+    "compatibility": [
+      {
+        "version": "2.0.0"
+      },
+      {
+        "version": "1.4.0",
```

Comment:
I'm not sure, but I think this should be `1.3.0`, see https://github.com/applitools/netlify-plugin-visual-diff/pull/8#issuecomment-865968498

Reply:
fixed!